### PR TITLE
Add structure factor to materials table

### PR DESCRIPTION
**Pull request by psavery**
This adds a column for the structure factor.

image

This is working, but I'm going to look into allowing for sorting the columns by structure factor.

Fixes: #703

### CODE REVIEW

**Comment by joelvbernier** [comment]
one more column -- multiplicity:

`PlaneData.getMultiplicity()`

**Comment by psavery** [comment]
> one more column -- multiplicity:
> 
> `PlaneData.getMultiplicity()`

Header will be `Multiplicity` ?

**Comment by joelvbernier** [comment]
yep

**Comment by psavery** [comment]
This is getting pretty close. Multiplicity has been added and sorting seems to be working. Just gonna do a little more testing first...

https://user-images.githubusercontent.com/9558430/108276298-34efb300-713d-11eb-99db-a5ba8d5af703.mp4

**Comment by psavery** [comment]
This is ready for review

**Comment by joelvbernier** [comment]
RAD!  LGTM

File: hexrd/ui/utils.py [hunk 1]
```diff
@@ -244,3 +245,13 @@ def __init__(self, function):
     def __get__(self, obj, type=None) -> object:
         obj.__dict__[self.name] = self.function(obj)
         return obj.__dict__[self.name]
+
+
+@contextmanager
+def exclusions_off(plane_data):
+    prev = plane_data.exclusions
+    plane_data.exclusions = [False] * len(plane_data.exclusions)
```
Comment (joelvbernier) [inline hexrd/ui/utils.py:253]: @psavery , setting `plane_data.exclusions = None` achieves the same result and may be "cleaner".

Comment (psavery) [reply hexrd/ui/utils.py:253]: Sounds good, I'll switch it.

**Review by joelvbernier — APPROVED** [review]

**Pull request merged**